### DISCUSSION TITLE
Exemplars in HTTP server metrics

### DISCUSSION THREAD

**Discussion by akselleirv** [open-telemetry/opentelemetry-java-instrumentation#9756] [Q&A] [answered]
Hello!

I'm converting OpenTelemetry metrics to Prometheus metrics using Grafana Agent, and one of the metrics I'm using is `http_server_duration_milliseconds_bucket`. I've checked that Grafana Agent supports converting exemplars, so it seems that the library instrumentation does not have any exemplars.

Is this the case? If so, would exemplars be added in the future?

**Comment by trask** [comment, marked as answer]
I believe exemplars for http server duration metric should be working, see tests and prior issues: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20exemplars&type=code

**Reply by akselleirv** [reply]
They do have exemplars when I test locally with: 
```
java -javaagent:opentelemetry-javaagent.jar -Dotel.metrics.exporter=prometheus -jar target/my-app.jar
```

And use curl to get the exemplars:

```
curl -H 'Accept: application/openmetrics-text' http://localhost:9464
```

So it might be that the Grafana Agent is the problem.
